### PR TITLE
Fix add user admin form is not display when toggle "Assign Current User as Administrator" to off

### LINKED ISSUE #1054: When add new site and Assign Current User as Administrator is disabled, form to add admin user is not displayed
## Description of bug
When add new site and Assign Current User as Administrator is disabled, form to add admin user is not displayed. The consequence is user can't create the new site. 

## Steps to reproduce
List the steps to reproduce the behavior:
1. Go to Persona Bar > Manage > Sites 
2. Click on Add Site 
3. Enter Site Title, Description, Select Site Template, Site Type as Directory, Set "Assign Current User as Administrator" toggle to off

## Current result
Form to add admin user is not displayed. 

## Expected result
Form to add admin user should displayed.

## Screenshots
If applicable, provide screenshots to help explain your problem.
image

## Error log
Paste the error log that is related to this issue.

## Additional context
Add any other context about the bug that may be helpful for resolution.

## Affected version

* [x] 9.3.2
* [x] 9.3.1
* [x] 9.2.2
* [x] 9.2.1
* [x] 9.2
* [x] 9.1.1
* [ ] 9.1
* [ ] 9.0

## Affected browser

* [x] Chrome
* [x] Firefox
* [x] Safari
* [x] Internet Explorer
* [x] Edge

### PR DESCRIPTION
**Pull request by tingung**
Fixes #1054 

## Summary

- Add fixedHeight to fix the issue 

Observation: GridSystem and GridCell within Collapsible required to set fixedHeight because height calculation is always zero. 

Confirmation video: https://drive.google.com/open?id=11XOMzx8Yrqq3i-TBxjoSQeEuphwdCeXc

### CODE REVIEW

**Review by valadas — APPROVED** [review]

**Comment by valadas** [comment]
Thanks @tingung that was quick :)

**Comment by tingung** [comment]
I have fixed the issue locally before creating the issue.

**Comment by v-karbovnichy** [comment]
@mitchelsellers @daguiler can you please check this?
@tingung needs approval and merge on this :)

**Comment by v-karbovnichy** [comment]
as far as I understand, we need to wait for `9.4.0` to be released before this merge

**Comment by valadas** [comment]
@tingung  this is currently targetting development which means it would be included in Dnn 10, would you like to re-target the release/3.0.x branch for inclusion in Dnn 9.4.1 ?

**Review by mitchelsellers — APPROVED** [review]

**Comment by valadas** [comment]
This got fixed in another way in #1098

**Pull request closed without merging**
